### PR TITLE
nginx - increase max upload file size to 100MB

### PR DESCRIPTION
**Pull request by pmauduit**

### CODE REVIEW

**Comment by pmauduit** [comment]
Tested into vagrant ; without it, it is not possible to send a 2.7MB file to the Datafeeder.

File: roles/nginx/templates/vhost.j2 [hunk 1]
```diff
@@ -39,6 +39,8 @@ server {
 
 	root /var/www/georchestra/htdocs/;
 
+  client_max_body_size 100M;
```
Comment (fvanderbiest) [inline roles/nginx/templates/vhost.j2:42]: weird indent ?

**Comment by pmauduit** [comment]
fixed ; my text editor ignored the fact that the file was using a mix of spaces and tabs.

File: roles/nginx/templates/vhost.j2 (hunk 1 above)
Comment (pmauduit) [reply roles/nginx/templates/vhost.j2]: fixed ; my text editor ignored the fact that the file was using a mix of spaces and tabs.

**Comment by landryb** [comment]
how about ... making it a playbook variable defaulting to 100M ? this way admins can tweak it easily ;)

**Comment by landryb** [comment]
> how about ... making it a playbook variable defaulting to 100M ? this way admins can tweak it easily ;)

done with 9bfa6d8

**Comment by pmauduit** [comment]
> how about ... making it a playbook variable defaulting to 100M ? this way admins can tweak it easily ;)

yes makes sense, sorry I missed the comment

**Pull request merged**